### ISSUE TITLE
thutessentials permissions with LuckPerms

### ISSUE THREAD

**Issue by smmmadden** [Thutmose/ThutEssentials#20]
Hi Thutmose, I'm trying to use the essentials.commands.land.* commands but they all get ignored as invalid commands.  I'm using LuckPerms Sponge version 4.0.62 (latest) with your most recent versions of the thutessentials, pokecube, revival etc.
I can see the perms fine for the other mods but not for thutessentials, thutbling or thutwearables.  The last two are probably fine if there are none, but all the permission classes from your project page have been added but they are not recognized by the permission mod.  Do you need to add support for this or other permissions mods so they get registered on startup of the server?

Is there something locally I need to do to enable them as a temp solution?
![image](https://user-images.githubusercontent.com/10309346/34331859-9a171992-e8f8-11e7-91c7-d267c44d0479.png)
![image](https://user-images.githubusercontent.com/10309346/34331865-ac39ee56-e8f8-11e7-9042-29fc53b842dc.png)

Checking the loaded mods that Sponge registered, here is the list:
![image](https://user-images.githubusercontent.com/10309346/34331971-93225dbc-e8f9-11e7-8c15-9a0190088800.png)
![image](https://user-images.githubusercontent.com/10309346/34331972-9d5c57c4-e8f9-11e7-8e3d-7277d7dc96b5.png)
It looks like the class is thut.essentials.ThutEssentials but I tried that and nothing shows.  Your website shows thut.essentials.commands so it isn't clear which one should be used. 
![image](https://user-images.githubusercontent.com/10309346/34332041-427341c8-e8fa-11e7-83b5-37a52ccdda50.png)
https://minecraft.curseforge.com/projects/thut-essentials

Thanks - Steve

**Comment by Thutmose** [comment]
The land classes specifically are disabled if you don't enable land in the configs, land is off by default as other land systems are normally more user friendly.

Do none of the commands show up in luckyperms? Not even say the /home, /spawn, etc?

**Comment by Thutmose** [comment]
That thut.essentials.ThutEssentials is the main mod class rather than anything actually related to permissions and commands

**Comment by smmmadden** [comment]
That is correct, none of the essentials permission nodes are shown in luckperms.
Now that I enabled both the land and teams options in the config, I can type in the commands and they are recognized, though many that require a playername or argument throw Java.lang.ArrayIndexOutOfBoundsExceptions:0 messages in-game and in-console a full traceback exception of the same.  There doesn't appear to be any traps to simply show the command usage help.
Typing in /help teaminvite returns: /teamInvite and not the actual text it should.

**Comment by smmmadden** [comment]
just to be clear, if ou type in /teaminvite without anything afterwards, it throws an exception.  It is more appropriate to display invalid command usage or something on those lines and then show how to use it.
Getting help on each command returns only the command in the chat window without the formatting of the command the user needs to see.

**Comment by InnuendoSA** [comment]
Sponge perm. plugins does not pick up forge perms. 
All so the other way around. 
Forge perm. mods does not pickup Sponge plugins perms.
